packages/fxa-payments-server/src/components/TermsAndPrivacy/index.tsx

```diff
@@ -27,6 +28,10 @@ export const TermsAndPrivacy = ({ plan }: TermsAndPrivacyProps) => {
     ? productDetailsFromPlan(plan, navigatorLanguages)
     : DEFAULT_PRODUCT_DETAILS;
 
+  const tosUrl = termsOfServiceDownloadURL
+    ? legalDocsRedirectUrl(termsOfServiceDownloadURL)
+    : termsOfServiceDownloadURL;
```

Comment:
It's because all the properties in the `ProductDetails` type are optional.

> Also, do we want to update DEFAULT_PRODUCT_DETAILS ...

No, that's fallback for missing metadata, and we shouldn't directly use the endpoint in the metadata.  The user would still get links that use the endpoint because of line 32 above and the planned email changes.